lib/mockbuilder.py

```diff
@@ -148,7 +148,8 @@ def _install_external_dependencies(self, package):
 
     def _create_build_directory(self, package):
         self.build_dir = os.path.join(
-            os.getcwd(), 'build', self.timestamp, package.name)
+            os.path.abspath(CONF.get('default').get('work_dir')), 'build',
+            self.timestamp, package.name)
```

Comment:
About the commit message: I suggest a more direct language:
Place the build directory used by mock under work-dir